simple_history/admin.py

```diff
@@ -231,6 +233,54 @@ def content_type_model_cls(self):
         """Returns the ContentType model class."""
         return django_apps.get_model("contenttypes.contenttype")
 
+    def revert_disabled(self, request, obj=None):
+        """If `True`, hides the "Revert" button in the `submit_line.html` template."""
+        if getattr(settings, "SIMPLE_HISTORY_REVERT_DISABLED", False):
+            return True
+        elif self.has_view_history_permission(
+            request, obj
+        ) and not self.has_change_history_permission(request, obj):
+            return True
+        return False
+
+    def has_view_permission(self, request, obj=None):
+        return super().has_view_permission(request, obj)
+
+    def has_change_permission(self, request, obj=None):
+        return super().has_change_permission(request, obj)
+
+    def has_view_or_change_permission(self, request, obj=None):
+        return self.has_view_permission(request, obj) or self.has_change_permission(
+            request, obj
+        )
```

Comment:
I'm curious as to why you added these methods when they provide exactly the same implementation as the super class. Is it to communicate that this can be a good starting point for future changes, should the need ever arise? In that case, it might be a good idea to add some comments 🙂

(This doesn't affect my approval in any way, though)